### PR TITLE
Support multiple chains

### PR DESCRIPTION
**Pull request by dan13ram**
- [x] needs testing of all flows
- [x] add chain labels wherever appropriate (ex: on game cards & game pages)
- [x] validate current chainid when sending transactions everywhere

### CODE REVIEW

**Review by ECWireless — APPROVED** [review]
All looks good to me!

File: hooks/useGames.ts [hunk 1]
```diff
@@ -1,50 +1,99 @@
...
+    });
+
+    setGames(sortedGames);
+    setError(_error);
+    setLoading(false);
+  }, []);
+
+  useEffect(() => {
+    fetchGames();
+  }, [refreshCount]);
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply hooks/useGames.ts]: Added in b51d74181bbc0251e795cd7f47ff0d5e45d3f72e

**Comment by ECWireless** [comment]
@dan13ram testing now, and only issue I've found is that setting the root for an item fails because of an `invalid signature`

File: package.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
     "urql": "^4.0.5",
     "viem": "^1.16.3",
     "wagmi": "^1.4.3",
-    "web3.storage": "^4.5.5"
+    "web3.storage": "^4.5.5",
+    "wonka": "^6.3.4"
```
[the remark this answers is not in the document]
Comment (dan13ram) [reply package.json:42]: Can remove actually. I was trying out something but it didn't work.

**Pull request merged**